bowling-ball/Frame.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+namespace BowlingBall
+{
+    public class Frame
+    {
+        private int roll1;
+        private int roll2;
+        public bool isSpare = false;
+        public bool isStrike = false;
+        protected int bonusRoll;
+        public Frame(int roll1,int roll2)
+        {
+            this.roll1 = roll1;
+            this.roll2 = roll2;
+        }
+        public int Roll1 { get { return roll1; } }
```

Comment:
Why both Properties and private members???
you can simply use public int Roll1 {get; private set;}
When you use private set, that variable can only be set in constructor.. I think you are trying to get same behavior, right???